kafka-rest/src/main/java/io/confluent/kafkarest/v2/KafkaConsumerManager.java

```diff
@@ -503,6 +600,74 @@ private synchronized void updateExpiration(KafkaConsumerState state) {
     Consumer createConsumer(Properties props);
   }
 
+  private class ReadTaskState {
```

Comment:
nit: could be static? Should probably check all these inner classes to see if they can be static. The advantage is they avoid the reference to the parent class.